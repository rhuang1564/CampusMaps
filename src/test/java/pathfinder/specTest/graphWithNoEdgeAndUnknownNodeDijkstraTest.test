# Find a path to nonexistant node with Dijkstra's algorithm
# in a no edged graph

CreateGraph graph

AddNode graph A
AddNode graph B

FindPath graph A E

FindPath graph E A

FindPath graph E E
