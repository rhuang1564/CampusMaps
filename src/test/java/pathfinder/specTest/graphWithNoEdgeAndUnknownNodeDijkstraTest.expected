# Find a path to nonexistant node with Dijkstra's algorithm
# in a no edged graph

created graph graph

added node A to graph
added node B to graph

unknown node E

unknown node E

unknown node E
unknown node E